.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ "main" ]
     paths-ignore: [ '**.md' ]
-  pull_request:
+  pull_request_target:
```

Comment:
to allow builds from forks, maintainers need to approve the workflow run after reviewing the PR.

Reply:
removed this file for now, will test the behavior separately from fork repos